docs/medical-api/more-info/contribution.mdx

```diff
@@ -0,0 +1,237 @@
+    "birthDate": "1970-03-15",
+    "qualification": [
+      {
+        "code": {
+          "coding": [
+            {
+              "system": "http://terminology.hl7.org/CodeSystem/v2-0360",
+              "code": "MD",
+              "display": "Doctor of Medicine"
+            }
+          ]
+        },
+        "period": {
+          "start": "1995-05-01"
+        },
+        "issuer": {
+          "reference": "Organization/XXXXX-XXXXXX-XXXXXX-XXXXXX"
+        }
+      }
+    ]
+  },
+  {
+    "resourceType": "Organization",
+    "id": "XXXXX-XXXXXX-XXXXXX-XXXXXX",
+    "identifier": [
+      {
+        "system": "http://hl7.org/fhir/sid/us-npi",
+        "value": "1234567890"
+      }
+    ],
+    "active": true,
+    "type": [
+      {
+        "coding": [
+          {
+            "system": "http://terminology.hl7.org/CodeSystem/organization-type",
+            "code": "prov",
+            "display": "Healthcare Provider"
+          }
+        ]
+      }
+    ],
+    "name": "Example Medical Center",
+    "telecom": [
+      {
+        "system": "phone",
+        "value": "+1 (555) 987-6543",
+        "use": "work"
+      },
+      {
+        "system": "email",
+        "value": "info@examplemedicalcenter.com",
+        "use": "work"
+      }
+    ],
+    "address": [
+      {
+        "use": "work",
+        "type": "both",
+        "line": ["456 Hospital Avenue"],
+        "city": "Anytown",
+        "state": "CA",
+        "postalCode": "12345",
+        "country": "USA"
+      }
+    ],
+    "contact": [
+      {
+        "purpose": {
+          "coding": [
+            {
+              "system": "http://terminology.hl7.org/CodeSystem/contactentity-type",
+              "code": "ADMIN",
+              "display": "Administrative"
+            }
+          ]
+        },
+        "name": {
+          "use": "official",
+          "family": "Johnson",
+          "given": ["Sarah"],
+          "prefix": ["Ms"]
+        },
+        "telecom": [
+          {
+            "system": "phone",
+            "value": "+1 (555) 789-0123",
+            "use": "work"
+          },
+          {
+            "system": "email",
+            "value": "sarah.johnson@examplemedicalcenter.com",
+            "use": "work"
+          }
+        ]
+      }
+    ]
+  }
+]
+```
+
+## Tips when Contributing
+
+- All Person references must point to the Metriport patient ID the data is being upload for in this example it's `subject.reference` in the Encounter resource.
+- All other references must point to a resource that's available within the same bundle.
+- You can't upload the same resource with the same id twice in the same Bundle, this will result in a conflict error.
+- All resources must be valid FHIR R4 resources.
+- Anything that would be useful and is clinically relevant to another provider should be contributed.
+  - This includes, but is not limited to, diagnoses, treatments, medications, and any significant findings from the encounter.
```

Comment:
let's reword this to `Ultimately, anything that would be useful and is clinically....`